test/jdk/java/awt/Mixing/AWT_Mixing/GlassPaneOverlappingTestBase.java

```diff
@@ -123,23 +152,21 @@ public void run() {
             }
             Point lLoc = testedComponent.getLocationOnScreen();
             lLoc.translate(1, testedComponent.getPreferredSize().height + 1);
-
-            /* this is a workaround for certain jtreg(?) focus issue:
-               tests fail starting after failing mixing tests but always pass alone.
-             */
-            Util.waitForIdle(robot);
-            ancestorLoc.translate(isOel7orLater() ? 5 : f.getWidth() / 2 - 15, 2);
-            robot.mouseMove(ancestorLoc.x, ancestorLoc.y);
-            Util.waitForIdle(robot);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
-            robot.delay(50);
-            robot.mouseRelease(InputEvent.BUTTON1_MASK);
-            Util.waitForIdle(robot);
-
-            clickAndBlink(robot, lLoc);
+            try {
+                if (!latch.await(1, TimeUnit.SECONDS)) {
+                    throw new RuntimeException("Ancestor frame didn't receive focus");
+                }
+                clickAndBlink(robot, lLoc);
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+            }
             return wasLWClicked;
         } else {
+            latch.countDown();
             return true;
         }
     }
+    protected void cleanup(){
```

Comment:
```suggestion
    }

    @Override
    protected void cleanup(){
```

Add a blank line between methods, and add the `@Override` annotation.